src/pages/AdoptionForm/index.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import Container from "react-bootstrap/Container";
+import Form from "react-bootstrap/Form";
+import { Col, Button, Image } from "react-bootstrap";
+import { postAdoption } from "../../store/user/actions";
+import { useDispatch, useSelector } from "react-redux";
+import { selectDogDetails } from "../../store/viewDogDeatils/selectors";
+
+export default function AdoptionForm() {
+  const dispatch = useDispatch();
+
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [province, setProvince] = useState("");
+  const [postcode, setPostcode] = useState("");
+  const [country, setCountry] = useState("");
+
+  const dog = useSelector(selectDogDetails);
+
+  function submitForm(event) {
+    event.preventDefault();
+    const dogId = dog.id;
+
+    dispatch(
+      postAdoption(
+        name,
+        email,
+        phone,
+        address,
+        city,
+        province,
+        postcode,
+        country,
+        dogId
+      )
+    );
+    setName("");
+    setEmail("");
+    setPhone("");
+    setAddress("");
+    setCity("");
+    setProvince("");
+    setPostcode("");
+    setCountry("");
+  }
+
+  return (
+    <Container>
+      <Form as={Col} md={{ span: 6, offset: 3 }} className="mt-5">
+        <h1 className="mt-5 mb-5">Adoption Form</h1>
+        {dog.imageUrl ? (
+          <Image src={dog.imageUrl} alt="preview" thumbnail />
+        ) : null}
```

Comment:
when one of the returns of the ternary is `null` you can always rewrite it with an && or ||. in this case:
```
{dog.imageUrl && (ImageComponent)}
```